### PR TITLE
changed file extention to scss and updated the path for it

### PR DESCRIPTION
**Pull request by zdsd0019**
blehg blehf dfkljsdzihgkfdnzlg nzfhd dxlkghnxd lhdgx

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import logo from './logo.svg';
-import './App.css';
+import './App.scss';
```
Comment (qmya) [inline src/App.js:3]: Why did you delete this line

Comment (zdsd0019) [reply src/App.js:3]: because i changed the extension
